spec/unit/ssl/host_spec.rb

```diff
@@ -512,22 +512,21 @@ def base_json_comparison(result, json_hash)
     end
 
```

Comment:
I think these tests could be improved by mocking at the level of Puppet::Rest::Routes.get_certificate and checking that the correct content has been written to disk when using a cert or cert bundle.

Reply:
Don't know if it's worth it though

Reply:
I think it might be. I'll look into that.